src/NewsService.Business/EditNewsCommand.cs

```diff
@@ -65,20 +64,16 @@ public OperationResultResponse<bool> Execute(Guid newsId, JsonPatchDocument<Edit
 
       OperationResultResponse<bool> response = new();
 
-      JsonPatchDocument<DbNews> dbRequest = _mapper.Map(request);
-
-      response.Body = _repository.Edit(newsId, dbRequest);
+      response.Body = _repository.Edit(_repository.Get(newsId), _mapper.Map(request));
```

Comment:
why you use get here? and what happened if we don't have this news

Reply:
Checked the existence of news, if there is no news, we will make a bad request.